src/rulesets/oas2/__tests__/model-description.ts

```diff
@@ -1,54 +0,0 @@
-import { DiagnosticSeverity } from '@stoplight/types';
-import { RuleType, Spectral } from '../../../spectral';
-import * as ruleset from '../index.json';
-
-describe('definition-description', () => {
```

Comment:
Ditto, has it been migrated?

Reply:
Dropped by @philsturgeon (as described in #725)

![image](https://user-images.githubusercontent.com/92363/69707923-f6ff0d00-10fa-11ea-81da-8718426342bc.png)